platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -667,26 +678,14 @@ private void createLocalPlatforms(
                         .build();
                 appRunThreads[ownHostIndex] = appThread;
 
-                platformRunThreads[ownHostIndex] = new ThreadConfiguration(getStaticThreadManager())
-                        .setDaemon(false)
-                        .setPriority(Settings.getInstance().getThreadPriorityNonSync())
-                        .setNodeId((long) ownHostIndex)
-                        .setComponent(SwirldsPlatform.PLATFORM_THREAD_POOL_NAME)
-                        .setThreadName("platformRun")
-                        .setRunnable(() -> {
-                            platform.run();
-                            // When the SwirldMain quits, end the run() for this platform instance
-                            Uninterruptable.abortAndLogIfInterrupted(
-                                    appThread::join, "interrupted when waiting for app thread to terminate");
-                        })
-                        .build();
-
                 ownHostIndex++;
                 synchronized (getPlatforms()) {
                     getPlatforms().add(platform);
```

Comment:
Since you now return the collection of platforms is this getter still needed?

Reply:
The thread model of `Browser.java` is completely borked... it defines a lot of things statically. A big part of this problem is the way that the UI code (not production code!) is interwoven with production code. The collection modified by `getPlatforms().add(platform)` is only ever accessed by UI code.

In the interest of not accidentally breaking the UI stuff, I'd prefer if we saved cleanup of the Browser's thread safety mechanisms for a stand alone PR.